tiny_dnn/nodes.h

```diff
@@ -486,15 +487,21 @@ class graph : public nodes {
         }
 
         for (auto l : input_layers_) {
-            graph_traverse(l, [=](layer& l) {}, [&](edge& e) {
-                auto next = e.next();
-                serial_size_t head_index = e.prev()->next_port(e);
-
-                for (auto n : next) {
-                    serial_size_t tail_index = n->prev_port(e);
-                    gc.add_connection(node2id[e.prev()], node2id[n], head_index, tail_index);
+            graph_traverse(
+                l,
+                [=](layer& l) {
+                    CNN_UNREFERENCED_PARAMETER(l);
+                },
```

Comment:
No. I disable Visual Studio's Text Editor C/C++ Formatting options because I prefer to indent code by myself.

I prefer to use below format to reduce eye positioning but abandoned it because of 80 characters per line rule.

```
            graph_traverse(l,
                           [=](layer& l) {
                               CNN_UNREFERENCED_PARAMETER(l);
                           },
                           [&](edge& e) {
                               auto next = e.next();
                               serial_size_t head_index = e.prev()->next_port(e);

                               for (auto n : next) {
                                   serial_size_t tail_index = n->prev_port(e);
                                   gc.add_connection(node2id[e.prev()],
                                                     node2id[n],
                                                     head_index,
                                                     tail_index);
                               }
                           }
                          );
```